packages/hint/docs/user-guide/telemetry/summary.md

```diff
@@ -12,6 +12,7 @@ following information:
 
 * The configuration used.
 * Scan status of individual hints: `"passed"|"failed"|"fixing"|"fixed"`.
+* A log of days used over the past 28 days.
```

Comment:
What do you think about linking to https://github.com/webhintio/hint/issues/3056#issuecomment-538142964 here so people can look and confirm that we aren't using any type of id?

Reply:
Good idea - I'll add that.